app/lib/account/consent_backend.dart

```diff
@@ -37,8 +37,15 @@ class ConsentBackend {
 
   ConsentBackend(this._db);
 
+  /// Throws if [consentId] is malformed.
+  void validateConsent(String consentId) {
+    InvalidInputException.checkMatchPattern(
+        consentId, 'consentId', RegExp(r'^[a-zA-Z0-9]*$'));
```

Comment:
or add `InvalidInputException.checkUlid(value, name)` I imagine we'll use ulid's else where too.